### PR TITLE
add info about next-gen/gen2 (basecluster-based) clusters to README.md

### LINKED ISSUE #108: user documentation: next-gen cluster provisioning (those using base clusters)

### PR DESCRIPTION
**Pull request by bcle**
Fixes #108

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -627,10 +627,73 @@ This example updates a cluster to use a new profile `my-new-profile`:
...
+If you make subsequent changes to the base cluster, workload clusters originally created from it
+will automatically acquire the changes.
+
+Before a manifest directory can be used as a base cluster, it must first be "prepared" or "prepped"
+by Arlon. The "prep" phase makes minor changes to the directory and manifest to help Arlon deploy
+multiple copies of the cluster without naming conflicts.
+
+To determine if a git directory is eligible to serve as base cluster, run the `basecluster validategit` command:
+```
+arlon basecluster validategit --repo-url <repoUrl> --repo-path <pathToDirectory> [--repo-revision revision]
```
Comment (Rohitrajak1807) [inline README.md:649]: We should also add `--repo-alias` examples. I can work on a dedicated section for `repo-alias`, and update existing examples.

**Review by Rohitrajak1807 — CHANGES_REQUESTED** [review]
Just a small suggestion, otherwise looks great

File: README.md (hunk 1 above)
Comment (bcle) [reply README.md:649]: Thanks, forgot about that. Since you're the expert on this topic, please add this info in a separate PR. Thanks!

**Comment by Rohitrajak1807** [comment]
Will do

**Review by Rohitrajak1807 — APPROVED** [review]

**Pull request merged**